### PR TITLE
Updated Readme , swift and Objective c projects for the 3.2.1 SDK in zapp

### PR DESCRIPTION
**Pull request by liviur**
Updated to the current SDK with updated readme and working projects.

### CODE REVIEW

**Comment by liviur** [comment]
@kononenkoAnton , @lomudi AHEM!!!!
The travis is bullshit that is not building for ages and should be removed.

File: FeedDemo-ObjectiveC/FeedDemo-ObjectiveC/AppDelegate.m [hunk 1]
```diff
@@ -8,8 +8,8 @@
 
 #import "AppDelegate.h"
 
-#import <Applicaster/APApplicaster.h>
-#import <Applicaster/APTimelinesManager.h>
+@import ApplicasterSDK;
+#import <ApplicasterSDK/APTimelinesManager.h>
```
Comment (kononenkoAnton) [inline FeedDemo-ObjectiveC/FeedDemo-ObjectiveC/AppDelegate.m]: Why do we need this import? +@import ApplicasterSDK;
should be enaugh

File: FeedDemo-ObjectiveC/FeedDemo-ObjectiveC/Classes/APFeedDemoTableViewController.m [hunk 2]
```diff
@@ -8,10 +8,10 @@
 
 #import "APFeedDemoTableViewController.h"
 
-#import <Applicaster/APTimelinesManager.h>
-#import <Applicaster/APFeedTimeline.h>
-#import <Applicaster/APFeedEpisode.h>
-#import <APFeed2/APNewFeedManager.h>
+#import <ApplicasterSDK/APTimelinesManager.h>
```
Comment (kononenkoAnton) [inline FeedDemo-ObjectiveC/FeedDemo-ObjectiveC/Classes/APFeedDemoTableViewController.m]: can be used +@import ApplicasterSDK;
not problem just cleaner

**Comment by kononenkoAnton** [comment]
@liviur Reviewed, look ok.
If you want make it clear fix imports.
And please delete travis.yaml

**Comment by liviur** [comment]
I'll make the changes , remove the travis yaml and merge.
Thanks.

**Pull request merged**